roles/container_runtime/tasks/package_docker.yml

```diff
@@ -111,12 +111,12 @@
     regexp: '^OPTIONS=.*$'
     line: "OPTIONS='\
       {% if ansible_selinux.status | default(None) == 'enabled' and openshift_docker_selinux_enabled | default(true) | bool %} --selinux-enabled {% endif %} \
-      {% if openshift_docker_log_driver | bool %} --log-driver {{ openshift_docker_log_driver }}{% endif %} \
+      {% if openshift_docker_log_driver %} --log-driver {{ openshift_docker_log_driver }}{% endif %} \
```

Comment:
I imagine the most clear way write this is `if openshift_docker_log_driver is defined` ?

Reply:
@sdodson `openshift_docker_log_driver` is already defined and defaults to `False`

https://github.com/openshift/openshift-ansible/blob/master/roles/container_runtime/defaults/main.yml#L28